Source/DafnyLanguageServer/Workspace/TextDocumentLoader.cs

```diff
@@ -93,7 +93,7 @@ INotificationPublisher notificationPublisher
         statusPublisher.SendStatusNotification(textDocument, CompilationStatus.ParsingFailed);
         return CreateDocumentWithEmptySymbolTable(loggerFactory.CreateLogger<SymbolTable>(), textDocument,
           errorReporter.GetDiagnostics(textDocument.Uri), program,
-          wasResolved: true, loadCanceled: false);
+          wasResolved: false, loadCanceled: false);
```

Comment:
Oh I see why you can change it from true to false. I remember when I added this variable, it used to mean `resolutionFinished: true`, meaning the resolution phase was completed. But it looks like this field isn't used anywhere anymore so feel free to reuse it for your means.